### ISSUE TITLE
fixed email is never sent

### ISSUE THREAD

**Issue by mekantor** [CastawayLabs/cachet-monitor#78]
I saw no parsing errors for the custom template, but it is not sent when the component gets marked as "up." The investigating emails are always sent.

Could any of you please post working examples where the "fixed" template gets sent? I've just configured a basic HTTP request and monitoring it for 200, then rename a file to make it fail on demand.

**Comment by karanp24** [comment]
you're right i am also having same issue, fixes email are not going out. In the logs you see this line 
time="2017-10-29T22:29:18Z" level=warning msg="Resolving incident" fields.time="29/10/2017 22:29:18 UTC" monitor="Portal" time="29/10/2017 22:29:18 UTC"
Can anyone guide me how to fix this issue? 
Thanks
